src/GeositeFramework/plugins/layer_selector_v2/main.js

```diff
@@ -109,15 +107,36 @@ define([
                 return layerId;
             },
 
-            createLayerMenu: function(layerId) {
+            showLayerMenu: function(el) {
+                var $el = $(el),
+                    layerId = this.getClosestLayerId(el),
+                    $menu = this._createLayerMenu(layerId),
+                    $shadow = this._createLayerMenuShadow(),
+                    supportsOpacity = this.state.serviceSupportsOpacity(layerId),
+                    pos = $el.offset(),
+                    top = supportsOpacity ? pos.top : pos.top + 50;
```

Comment:
The positioning is a little off for layers that do not support opacity.

![image](https://cloud.githubusercontent.com/assets/43062/13058971/098e6c1e-d3f5-11e5-82f3-7d1cfeabd9c4.png)